persist/fs/commitlog/reader.go

```diff
@@ -501,7 +509,7 @@ func (r *reader) readInfo() (schema.LogInfo, error) {
 		return emptyLogInfo, err
 	}
 	data.IncRef()
-	r.logDecoder.Reset(data.Get())
+	r.logDecoder.Reset(encoding.NewDecoderStream(data.Get()))
```

Comment:
same here, should we pool `DecoderStream` somehow?

Reply:
I can reuse the decoderstream for the reader, sure thing.